File: go/problem_02.go
package main
import "fmt"


func Fibonacci(n int)  int{
  if n < 2 {
    return n
  } else {
    return Fibonacci(n-2) + Fibonacci(n-1)
  }
}

func main(){
  sum := 0
  fibo := 0
  for i:= 0; fibo <= 4000000; i++ {
    fibo = Fibonacci(i)
    if fibo % 2 == 0 {
      sum += fibo
    }
  }
  fmt.Println(sum)
}

File: python/problem_02.py
def Fibonacci(n):
    if n == 2 or n == 1:
        return 1
    else:
        return Fibonacci(n-1) + Fibonacci(n-2)


suma = 0
fibo = 0

for i in range(0, 4000000):
    fibo = Fibonacci(i)
    if fibo % 2 == 0:
        suma += fibo
    print(suma)
